content/it/docs/concepts/overview/kubernetes-api.md

```diff
@@ -0,0 +1,126 @@
+
+- Livello alpha:
+  - Il nome di versione contiene `alpha` (e.g. `v1alpha1`).
+  - Potrebbe contenere dei *bug*.  Abiliare questa funzionalità potrebbe esporre al rischio di *bugs*.  Disabilitata di default.
+  - Il supporto di questa funzionalità potrebbe essere rimosso in ogni momento senza previa notifica.
+  - Questa API potrebbe cambiare in modo incompatibile in rilasci futuri del Software e senza previa notifica.
+  - Se ne raccomandata l'utilizzo solo in *clusters* di test creati per un breve periodo di vita, a causa di potenziali *bugs* e delle mancanza di un supporto di lungo periodo.
+- Livello beta:
+  - Il nome di versione contiene `beta` (e.g. `v2beta3`).
+  - Il codice è propriamente testato.  Abilitare la funzionalità è considerato sicuro.  Abilitata di default.
+  - Il supporto per la funzionalità nel suo complesso non sarà rimosso, tuttavia potrebbe subire delle modifiche.
+  - Lo schema e/o la semantica delle risorse potrebbe cambiare in modo incompatibile in successivi rilasci beta o stabili. Nel caso questo dovesse verificarsi, verrano fornite istruzioni per la migrazione alla versione successiva. Questo potrebbe richiedere la cancellazione, modifica, e la ri-creazione degli oggetti supportati da questa API.  Questo processo di modifica potrebbe richiedere delle valutazioni.  La modifica potrebbe richiedere un periodo di non disponibilità dell'applicazione che utilizza questa funzionalità.
+  - Raccomandata solo per applicazioni non critiche per la vostra impresa a causa dei potenziali cambiamenti incompatibili in rilasci successivi. Se avete più *clusters* che possono essere aggiornati separatamente, potreste essere in grado di gestire meglio questa limitazione.
+  - **Per favore utilizzate le nostre versioni beta e forniteci riscontri relativamente ad esse!  Una volta promosse a stabili, potrebbe non essere semplice apportare cambiamenti successivi.**
+- Livello stabile:
+  - Il nome di versione è `vX` dove `X` è un intero.
+  - Le funzionalità relative alle versioni stabili continueranno ad essere presenti per parecchie versioni successive.
+
+## API groups
+
+Per facilitare l'estendibilità delle API di Kubernetes, sono stati implementati gli [*API groups*](https://git.k8s.io/community/contributors/design-proposals/api-machinery/api-group.md).  
+L'*API group* è specificato nel percorso REST ed anche nel campo `apiVersion` di un oggetto serializzato.
+
+Al momento ci sono diversi *API groups* in uso:
+
+1. Il guppo *core*, spesso referenziato come il *legacy group*, è disponibile al percorso REST `/api/v1` ed utilizza `apiVersion: v1`.
+
+1. I gruppi basati su un nome specifico sono disponibili attraverso il percorso REST `/apis/$GROUP_NAME/$VERSION`, ed usano `apiVersion: $GROUP_NAME/$VERSION` (e.g. `apiVersion: batch/v1`).  La lista completa degli *API groups* supportati e' descritta nel documento [Kubernetes API reference (EN)](/docs/reference/).
```

Comment:
There are two "1." points.
    Two spaces after the dot. (This is happening also in other lines but since it's really a minor thing, I don't feel it's necessary to fix them)